embeddings/pipeline/hps_pipeline.py

```diff
@@ -98,19 +109,15 @@ def get_best_paramaters(self, study: Study) -> Metadata:
         return self._get_metadata(parsed_params)
 
     def run(
-        self,
+        self, run_name: Optional[str] = None, catch: Tuple[Type[Exception], ...] = (), **kwargs: Any
```

Comment:
Imo, it should be empty for the development purposes, and can be filled in with proper exceptions at the user's risk. The previous configuration with `(Exception,)` was very inconvenient during my trials, because I was not able to find out where  is the problem.

Reply:
User will use various models, datasets which could be problematic.

Reply:
I get your motivation, but I do not agree with you.

Catching exception here changes behaviour of HPS instead of breaking whole optimilaztion pipeline, it logs error to given trial. 

Why it is desired behaviour?
1. Some configurations in given Config Space are unavailable to run on GPUs (and GPUs are our default use-case, and not testing or CPU runs) due to insufficient memory. Defining config spaces that are valid for every combination of parameters on GPU is almost impossible.
2. User can still done debugging as excpetions are logged in logs DataFrame. 
3. Users can make some test trials with like 5 runs to check whether there were sucessful.
4. Standard problems with models / datasets should be resolved in basic pipelines and not in hps stage.